JustBrewIt/Pages/Search.cshtml

```diff
@@ -1,35 +1,34 @@
 ﻿@page
 @model JustBrewIt.Pages.SearchModel
 @{
+
+    ViewData["Title"] = "Search Page";
```

Comment:
Fixed: Added naming for the pages   ViewData["Title"] = "Search Page";    in Search.cshtml page